### PR TITLE
Gh actions

### PR DESCRIPTION
**Pull request by micson-us**
Realized coverage test step added 3 months ago failed so updated the yml file from main and confirmed success.

### CODE REVIEW

File: .github/workflows/cd-workflow.yml [hunk 1]
```diff
@@ -3,13 +3,55 @@ name: Docker CD
 on:
   push:
     branches: 
-      - feature/regration_testing
+      - main
```
Comment (mattdrayer) [inline .github/workflows/cd-workflow.yml:6]: So this action will trigger upon merge to the `main` branch, yes?

Comment (KarenAJ) [reply .github/workflows/cd-workflow.yml:6]: @micson-us

Comment (micson-us) [reply .github/workflows/cd-workflow.yml:6]: that is correct

**Review by KarenAJ — APPROVED** [review]
Approved

**Review by SHGODE — APPROVED** [review]
Approved

**Pull request merged**